### DISCUSSION TITLE
Roadmap and adoption

### DISCUSSION THREAD

**Discussion by brno32** [jonathan-s/django-sockpuppet#100] [General]
I've been taking a deep look at this framework and I think it looks awesome!

I'm currently a React developer, and while I enjoy it, I do miss the velocity with which you can work when using Django's templating, and it can be tedious building APIs for an app's internal usage. Being able to leverage all of Django's wonderful features (server-side state management, auth, middleware, cookie management, etc) while still building a snappy frontend looks real appealing!

I'm curious what the outlook for this project is. I want to try it out, but want to have a better idea of what its future looks like before implementing it in a job for a client. I thought it'd be neat to start this discussion here.

**Comment by jonathan-s** [comment]
Hey @brno32, 
Sorry for the late reply here, seems like I missed the notification. The plan is to keep the feature-set in line with the feature development happening in StimulusReflex (that's for rails) as it builds on the same frontend part. I'm also happy to receive suggestions on how things can improve of course and we can work that into the library.
